src/utils/formValidation.ts

```diff
@@ -1,6 +1,9 @@
 export const mustBePresent = (value: any): string | undefined =>
   value ? undefined : 'validation.message.required';
 
+export const mustHaveSelection = (value: any): string | undefined =>
+  value && value !== '-' ? undefined : 'validation.message.required';
+
```

Comment:
Would make sense to add a falsy value like `<option value=''>-</option>` in https://github.com/City-of-Helsinki/berth-reservations-ui/blob/91cb1353502f63d10698cf6e796f565c13ef0bc3/src/components/forms/Selects.tsx#L75 and other similar selects
Instead of having this other validator?
Because changing `-` will break this validator.

Reply:
Reverted back and changed the values of all empty selections as suggested here.